### ISSUE TITLE
May you help to add a function like this? Thanks!

### ISSUE THREAD

**Issue by nanjingman** [dominiclet/obsidian-note-definitions#65]
![image](https://github.com/user-attachments/assets/3f92bfc8-0983-4101-a967-c8d177ce1ae4)

**Comment by dominiclet** [comment]
Are you referring to adding multiple definition folders? As mentioned in issue #52?

**Comment by nanjingman** [comment]
Yes, I think it can offer more flexbility. this is a sample from note toolbar. 
![image](https://github.com/user-attachments/assets/0d080a3a-0f5d-4b9d-814c-7c6806c5acb5)

**Comment by nanjingman** [comment]
Yes, I think it can offer more flexbility.

**Comment by Riven53** [comment]
It would be very useful but could become very slow when adding definitions to a file in a specific folder when there a many folders and files for definitions.

Having two of these options 
![image](https://github.com/user-attachments/assets/191a1c0c-229e-42a1-8ae0-39f46be288dd)
and scrolling thru both of them is quiet time consuming.

but when there aren't that many folders and files it becomes very very convenient

**Comment by nanjingman** [comment]
Thanks! I think using the "atomic" definition option is already OK!
